### DISCUSSION TITLE
Predefined queries and mutations

### DISCUSSION THREAD

**Discussion by jaulz** [nuwave/lighthouse#2561] [Q&A]
Is it possible to configure predefined queries and mutations so no other queries and mutations are possible? That would allow me to limit the attack surface.

**Comment by spawnia** [comment]
No, but it should be possible to reuse the facilities established for [Automatic Persisted Queries](https://lighthouse-php.com/master/performance/query-caching.html#automated-persisted-queries) to achieve such a feature.
